models/user.go

```diff
@@ -65,6 +72,17 @@ func (d *UserData) Value() interface{} {
 		return d.NumericValue
 	case BOOL:
 		return d.BoolValue
+	case MAP:
+		data := &map[string]interface{}{}
+		json.Unmarshal([]byte(d.SerializedValue), data)
+		return data
+	case ARRAY:
```

Comment:
Merging in for now to get app_metadata working - but we will need to revisit this